### DISCUSSION TITLE
FlexBe VS BehaviorTree.CPP

### DISCUSSION THREAD

**Discussion by skyrobos** [BehaviorTree/BehaviorTree.CPP#670] [Q&A]
what are the major differences between these? except for the programming language

**Comment by facontidavide** [comment]
I don't know FlexBe enough to answer.

What you MAY find unique about this library is:

- Support for a scripting language on top of the Behavior tree common syntax.
- Groot2: a gui to edit and monitor your behavior trees
- A lot of utilities to debug, log and introspect your behaviors.
